vue-components/package.json

```diff
@@ -33,8 +33,10 @@
   ],
   "types": "dist/main.d.ts",
   "dependencies": {
+    "@types/lodash": "^4.14.164",
```

Comment:
These should probably be in the dev dependencies

Reply:
true!